packages/primeng/src/dialog/style/dialogstyle.ts

```diff
@@ -18,6 +18,7 @@ const theme = ({ dt }) => `
 .p-dialog-content {
     overflow-y: auto;
     padding: ${dt('dialog.content.padding')};
+    flex: 1 1 auto
```

Comment:
Hi, have you tried with   `flex-grow: 1;` instead of this, it is present in v17 but missing in this. also semicolon is missing.

Reply:
I have  implemented the changes you suggested to be consistent with previous versions of PrimeNG.  I have confirmed the change with the original reproducer I provided in the  issue.

Note:  both **flex: 1 1 auto** and **flex-grow: 1**  and (I also tested **flex: auto**) all produced the same  positive results